net/libtorrent-devel/Portfile

```diff
@@ -49,8 +49,19 @@ post-destroot {
         ${destroot}${docdir}
 }
 
-# FIXME: 32-bit systems need to switch to C++11 atomics instead of __sync* builtins:
 # https://github.com/rakshasa/libtorrent/issues/163
+if {[string match macports-gcc* ${configure.compiler}]} {
+    patchfiles-append \
+                    0001-Fix-atomics-for-modern-gcc.patch
+
+    # thread_base.cc: error: invalid conversion from 'const char*' to 'char*' [-fpermissive]
+    configure.cxxflags-append \
+                    -fpermissive
```

Comment:
Why do that instead of actually fixing the problem or putting a coercion into the code?

Reply:
Cannot check at the moment, but I think this is SDK-related. We cannot control SDK headers.